fgpyo/sam/__init__.py

```diff
@@ -590,6 +587,70 @@ def set_pair_info(r1: AlignedSegment, r2: AlignedSegment, proper_pair: bool = Tr
     r2.template_length = -insert_size
 
 
+@attr.s(auto_attribs=True, frozen=True)
+class ReadEditInfo:
+    """Counts various stats about how a read compares to a reference sequence."""
+
+    mismatches: int
+    insertions: int
```

Comment:
consider stating somewhere the # of insertions are relative to the reference (i.e. extra bases in the read that are not in the reference).  I am going to come back to this in 6mo and not remember and have to look at the code.

Reply:
Lol, you've been spending too much time in gene editing space ;)  I'll add docs.